src/Common/src/Common/Util/MethodInvoker.cs

```diff
@@ -168,7 +166,9 @@ protected virtual MethodInfo FindMatchingMethod()
                 throw new InvalidOperationException("No target class set");
             }
 
+#pragma warning disable S3011 // Reflection should not be used to increase accessibility of classes, methods, or fields
```

Comment:
Thoughts on this rule and resolving vs disabling?

Reply:
Disable globally ... when writing framework code we will at times need to implement code that reflects on and invokes private, internal, protected, etc, methods, fields, properties, ...

Reply:
Thanks Dave!